cves/CVE-2017-7659.yml

```diff
@@ -139,8 +147,11 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
+    - commit: 29f50cf567423ca354ee0d5fa0dfb015c555cf55
+      note: |
```

Comment:
I agree with you, that does seem interesting. Was there any rationale in the commit message or the comments for why they replaced the module?